docs/content/examples/tutorials/basic-installation.md

```diff
@@ -84,142 +79,3 @@ services:
       - NET_ADMIN # For Fail2Ban to work
     restart: always
 ```
-
-## A Detailed Example
-
-!!! note
-
-    This is a community contributed guide. Please let us know via a Github Issue if you're having any difficulty following the guide so that we can update it.
-
-This guide is focused on only using [SMTP ports (not POP3 and IMAP)][docs-ports] with the intent to send received mail to another MTA service such as _Gmail_. It is not intended to have a MUA client (_eg: Thunderbird_) to retrieve mail directly from `docker-mailserver` via POP3/IMAP.
-
-In this setup `docker-mailserver` is not intended to receive email externally, so no anti-spam or anti-virus software is needed, making the service lighter to run.
-
-!!! warning "Open Relays"
-
-    Adding the docker network's gateway to the list of trusted hosts (_eg: using the `network` or `connected-networks` option_), can create an [**open relay**](https://en.wikipedia.org/wiki/Open_mail_relay). For instance [if IPv6 is enabled on the host machine, but not in Docker][github-issue-1405-comment].
```

Comment:
Related to the ENV in the compose config example in this guide, this makes the concern a bit more apparent up front. We do cover this in the ENV docs and the FAQ entry also includes the referenced github issue, so not a major loss, but could be useful to include in `usage.md` if that will be a more official guide?

Reply:
It's good you re-added it. We can keep it there.